code_samples/psi_demo/README.md

```diff
@@ -0,0 +1,28 @@
+# PSI Demo [![JetBrains IntelliJ Platform SDK Docs](https://jb.gg/badges/docs.svg)][docs]
+*Reference: [Navigating the PSI in IntelliJ SDK Docs][docs:navigating_psi]*
+
+## Quickstart
+
+PSI Demo project demonstrates working with the PSI Navigation by implementing `AnAction` that through the message
+dialog, informs about:
```

Comment:
Suggest: _The PSI Demo project shows exploring PSI Navigation by implementing `AnAction` that explores a `PsiFile` associated with a file open in the editor. Using the plugin's action and message dialog, a user can move the editor caret and see information about:_